app/src/main/java/xyz/kotlout/kotlout/model/adapter/ExperimentListAdapter.java

```diff
@@ -252,42 +251,28 @@ public View getChildView(int groupPosition, int childPosition, boolean isLastChi
     TextView region = convertView.findViewById(R.id.tv_experiment_list_region);
     TextView counter = convertView.findViewById(R.id.tv_experiment_list_counter);
     TextView type = convertView.findViewById(R.id.tv_experiment_list_type);
+    ImageView icon = convertView.findViewById(R.id.iv_experiment_list_visible);
 
     ExperimentGroup experimentGroup = ExperimentGroup.getByOrder(groupPosition);
     ExperimentController experimentController = experimentGroups.get(experimentGroup).get(childPosition);
 
+    if (listType == ListType.ALL || listType == ListType.SUBSCRIBED) {
+      icon.setVisibility(View.GONE);
+    } else {
```

Comment:
`else if { ... }` instead of `else { if { ... } }` ?

Reply:
Changed it to a switch, also fixed a bug